assets/python/processing/processing:find_extrema.pyi

```diff
@@ -0,0 +1,31 @@
+# ---------------------------------------------------------
+# This script aims to determine extrema for a given array.
+# Please adjust the parameters according to your data.
+# ---------------------------------------------------------
+import numpy as np
+from scipy.signal import find_peaks
+import json
+from munch import Munch
+
+# Data From Context
+# -----------------
+# The array can be extracted from the context by rendering the following Jinja template
```

Comment:
I'd move this comment to the top to make it clear that `array_from_context` is the variable expected.

We should also add example of what's inside the array - is that an array of tuples or just one-dimensinal array?